tn_api_spec.yml

```diff
@@ -47,6 +47,32 @@ info:
     response = requests.post(url, headers=headers, data=json.dumps(payload))
     </pre>
 
```

Comment:
Should we have the error code list after introducing FareStructure and its req/rsp formats?